src/wallet/wallet.service.ts

```diff
@@ -139,17 +152,20 @@ export class WalletService implements OnModuleInit {
 
   /**
    * Signs a message to pause deposits
-   * @param prefix - unique prefix from the contract for this type of message
-   * @param blockNumber - block number that is signed
+   * @param signPauseDataParams - parameters for signing pause message
+   * @param signPauseDataParams.prefix - unique prefix from the contract for this type of message
+   * @param signPauseDataParams.blockNumber - block number that is signed
+   * @param signPauseDataParams.stakingModuleId - target staking module id
    * @returns signature
    */
-  public async signPauseData(
-    prefix: string,
-    blockNumber: number,
-  ): Promise<Signature> {
+  public async signPauseData({
+    prefix,
+    blockNumber,
+    stakingModuleId,
+  }: SignPauseDataParams): Promise<Signature> {
     const encodedData = defaultAbiCoder.encode(
-      ['bytes32', 'uint256'],
-      [prefix, blockNumber],
+      ['bytes32', 'uint256', 'uint24'],
+      [prefix, blockNumber, stakingModuleId],
```

Comment:
stakingModuleId  is `uint256` according to the current contract code
https://github.com/lidofinance/lido-dao/blob/feature/shapella-upgrade/contracts/0.8.9/DepositSecurityModule.sol#L307-L311